includes/elements/column.php

```diff
@@ -298,7 +298,8 @@ protected function register_controls() {
 						'background_lazyload' => [
 							'active' => true,
 							'keys' => [ 'background_image', 'url' ],
-							'selector' => '.elementor-column-wrap',
+							'overlay_keys' => [ 'background_overlay_image', 'url' ],
```

Comment:
the overlay is not a different control? lets talk about it

Reply:
Keys are removed, using new logic to support multiple background controls in the same element.